understanding/20/focus-order.html

```diff
@@ -54,6 +54,31 @@ <h2>Intent of Focus Order</h2>
          Web page. Care should be taken so that the focus order makes sense to both of these
          sets of users and does not appear to either of them to jump around randomly. 
       </p>
+
+      <p>Focus order also does not normatively need to follow the visual presentation of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, visually the assumption in western left-to-right systems may be that elements in the
+         left-hand column would receive focus first. However, if meaning and operation are not affected,
+         it is not a normative failure if elements in the right-hand column receive focus first.
+         Again, as a best practice, care should be taken to make sure the focus order does not
+         appear random and illogical.
+      </p>
+
+      <div class="note">
+         <p>Normatively, it is not a failure of this success criterion if static/non-interactive content
+            receives focus. Making an element that does not act as an actionable user interface component
+            focusable, or programmatically moving focus to such an element, does not necessarily constitute
+            a failure of this success criterion. Similarly, redundant focusable elements – for instance,
+            having two or more "focus stops" for what may visually appear to be a single user interface
```

Comment:
might be useful for a clarifying example here, especially since the example of nested focusable elements is used later as a possible mean of failure.

regarding an example that I would say falls into the "not a failure but annoying" category would be a "card" like pattern.  where an image and heading text are both separate links, but could/should have been a single link.

Reply:
expanded this a little bit. danger is adding too much HTML in understanding, as it's supposed to be slightly more "tech agnostic" ...

Reply:
understood, but i think the addition is helpful.  thank yoiu